projects/ngrx.io/content/guide/component/let.md

```diff
@@ -5,18 +5,18 @@ It also helps with several internal processing under the hood.
 
 The current way of binding an observable to the view looks like that:
 ```html
-<ng-container *ngIf="observableNumber$ as n">
+<ng-container *ngIf="observableNumber$ | async as n">
   <app-number [number]="n">
   </app-number>
   <app-number-special [number]="n">
   </app-number-special>
 </ng-container>
  ```
 
-The problem is `*ngIf` is also interfering with rendering and in case of a `0` the component would be hidden
+The problem is `*ngIf` is also interfering with rendering and in case of a falsy value the component would be hidden
 
 The `*ngrxLet` directive take over several things and makes it more convenient and save to work with streams in the template
-`<ng-container *let="observableNumber$ as c"></ng-container>`
+`<ng-container *ngrxLet="observableNumber$ as c"></ng-container>`
```

Comment:
Thanks for spotting this!
I did a find, and also saw `*let` was used in the TS docs, do you mind changing that too please?

https://github.com/ngrx/platform/blob/master/modules/component/src/let/let.directive.ts#L80

Reply:
Done! Thank you for pointing it out.